composer.json

```diff
@@ -1,33 +1,41 @@
 {
-    "name": "drift/skeleton",
-    "description": "Skeleton repository ofr DriftPHP",
-    "type": "project",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Marc Morera",
-            "email": "yuhu@mmoreram.com"
-        }
-    ],
-    "require": {
-        "php": "^7.3",
-        "symfony/console": "^5.0.0",
-        "symfony/framework-bundle": "^5.0.0",
-        "symfony/dotenv": "^5.0.0",
-        "symfony/stopwatch": "^5.0.0",
-        "symfony/debug": "^4.4.0",
-
-        "drift/http-kernel": "0.1.*, >=0.1.7",
-        "drift/server": "0.1.*, >=0.1.7",
-        "drift/react-functions": "0.1.*",
-        "drift/event-loop-utils": "0.1.*"
-    },
-    "autoload": {
-        "psr-4": {
-            "App\\": "src/",
-            "Domain\\": "src/Domain",
-            "Infrastructure\\": "src",
-            "Drift\\": "Drift/"
-        }
+  "name": "drift/skeleton",
+  "description": "Skeleton repository ofr DriftPHP",
+  "type": "project",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Marc Morera",
+      "email": "yuhu@mmoreram.com"
     }
-}
\ No newline at end of file
+  ],
+  "require": {
+    "php": "^7.3",
+    "symfony/console": "^5.0.0",
+    "symfony/framework-bundle": "^5.0.0",
+    "symfony/dotenv": "^5.0.0",
+    "symfony/stopwatch": "^5.0.0",
+    "symfony/debug": "^4.4.0",
+    "drift/http-kernel": "0.1.*, >=0.1.7",
+    "drift/server": "0.1.*, >=0.1.7",
+    "drift/react-functions": "0.1.*",
+    "drift/event-loop-utils": "0.1.*",
+    "symfony/flex": "^1.8"
+  },
+  "autoload": {
+    "psr-4": {
+      "App\\": "src/",
+      "Domain\\": "src/Domain",
+      "Infrastructure\\": "src/Infrastructure",
+      "Drift\\": "Drift/"
+    }
+  },
+  "scripts": {
+    "server": "server run 0.0.0.0:8000",
```

Comment:
Did this work in your local? I mean. How could this work? `server run` is not a kernel-loaded command, but an isolated one. Same for watcher.

Reply:
Quote [https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands](composer docs): 

> Before executing scripts, Composer's bin-dir is temporarily pushed on top of the PATH environment variable so that binaries of dependencies are easily accessible. In this example no matter if the phpunit binary is actually in vendor/bin/phpunit or bin/phpunit it will be found and executed.

So this works perfectly on each environment, since its only a convenient way of executing scripts in `vendor/bin`

Reply:
Let me try again. The last time it didn't work properly for me.